network/network_test.go

```diff
@@ -175,7 +175,7 @@ func newTestNetwork(t *testing.T, count int) (*testDialer, []*testListener, []id
 		require.NoError(t, err)
 		nodeID := ids.NodeIDFromCert(cert)
 
-		blsKey, err := bls.NewSecretKey()
+		blsKey, err := bls.NewSigner()
```

Comment:
🤔 If we want to change all the `*Key` names (variables or properties), I would recommend/request that I do that in a follow-up to limit the scope of both changes.